contrib/babelfishpg_tsql/src/collation.c

```diff
@@ -79,10 +80,15 @@ collation_list(PG_FUNCTION_ARGS)
 	PG_RETURN_DATUM(tsql_collation_list_internal(fcinfo));
 }
 
+
+/*
+ * get_server_collation_oid - this is being used by get_babel_server_collation_oid_deprecated_in_2_3_0
+ * which is depcrecated and is not recommended to be used.
+ */
 Datum
 get_server_collation_oid(PG_FUNCTION_ARGS)
 {
-	PG_RETURN_OID(tsql_get_server_collation_oid_internal(false));
+	PG_RETURN_OID(InvalidOid);
```

Comment:
Let's add a warning message (reference: https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_2_X_DEV/contrib/babelfishpg_tsql/src/rolecmds.c#L1364)